ui/raidboss/data/06-ew/raid/p9n.txt

```diff
@@ -1,6 +1,364 @@
+300.0 "--sync--" sync / 1[56]:[^:]*:Kokytos:8123:/ # ice marker
+304.4 "Fire III (Static)" sync / 1[56]:[^:]*:Kokytos:811E:/
+304.4 "Blizzard III (Shrinking)" sync / 1[56]:[^:]*:Kokytos:8121:/
+304.8 "--sync--" sync / 1[56]:[^:]*:Kokytos:811C:/
+
+313.9 "Global Spell" sync / 1[56]:[^:]*:Kokytos:8141:/ jump 500.0
+317.1 "Disgorge" #sync / 1[56]:[^:]*:Kokytos:8124:/
+329.3 "Ravening (Fighter)" #sync / 1[56]:[^:]*:Kokytos:8119:/
+350.7 "Archaic Rockbreaker" #sync / 1[56]:[^:]*:Kokytos:8128:/
+
+# -> after Dualspell
+500.0 "Global Spell" sync / 1[56]:[^:]*:Kokytos:8141:/
+500.9 "--sync--" sync / 1[56]:[^:]*:Kokytos:8142:/
+503.2 "Disgorge" sync / 1[56]:[^:]*:Kokytos:8124:/
+507.2 "--sync--" sync / 1[56]:[^:]*:Kokytos:8115:/
+510.3 "--sync--" sync / 1[56]:[^:]*:Kokytos:8115:/
+
+
+
+# Fighter phase 1
+515.4 "Ravening (Fighter)" sync / 1[56]:[^:]*:Kokytos:8119:/
+524.5 "--sync--" sync / 1[56]:[^:]*:Kokytos:8127:/
+527.6 "--sync--" sync / 1[56]:[^:]*:Kokytos:8127:/
+530.7 "--sync--" sync / 1[56]:[^:]*:Kokytos:8127:/
+536.8 "Archaic Rockbreaker" sync / 1[56]:[^:]*:Kokytos:8128:/
+538.4 "Shockwave" sync / 1[56]:[^:]*:Kokytos:8129:/
+541.1 "--sync--" sync / 1[56]:[^:]*:Kokytos:8127:/
+544.2 "--sync--" sync / 1[56]:[^:]*:Kokytos:8127:/
+545.3 "Archaic Rockbreaker 1" sync / 1[56]:[^:]*:Kokytos:812A:/
+547.3 "--sync--" sync / 1[56]:[^:]*:Kokytos:8127:/
+550.3 "Archaic Rockbreaker 2" sync / 1[56]:[^:]*:Kokytos:812A:/
+556.9 "Archaic Demolish" sync / 1[56]:[^:]*:Kokytos:812F:/
+557.9 "--sync--" sync / 1[56]:[^:]*:Kokytos:8130:/
+564.0 "Gluttony's Augur" sync / 1[56]:[^:]*:Kokytos:8116:/
+564.4 "Gluttony's Augur" sync / 1[56]:[^:]*:Kokytos:8117:/
+571.1 "Ascendant Fist" sync / 1[56]:[^:]*:Kokytos:8131:/
+573.2 "--sync--" sync / 1[56]:[^:]*:Kokytos:8127:/
+576.3 "--sync--" sync / 1[56]:[^:]*:Kokytos:8127:/
+583.4 "Front Combination/Rear Combination" sync / 1[56]:[^:]*:Kokytos:814[8A]:/
+583.6 "Outside Roundhouse" sync / 1[56]:[^:]*:Kokytos:812C:/
+586.6 "Swinging Kick" sync / 1[56]:[^:]*:Kokytos:812[DE]:/
+590.7 "Disgorge" sync / 1[56]:[^:]*:Kokytos:8125:/
+594.7 "--sync--" sync / 1[56]:[^:]*:Kokytos:8115:/
+597.8 "--sync--" sync / 1[56]:[^:]*:Kokytos:8115:/
+599.0 "--sync--" sync / 1[56]:[^:]*:Kokytos:8144:/
+
+
+
+# Behemoth phase 1
+603.7 "Ravening (Behemoth)" sync / 1[56]:[^:]*:Kokytos:811A:/
+621.0 "Charybdis" sync / 1[56]:[^:]*:Kokytos:8132:/
+622.0 "--sync--" sync / 1[56]:[^:]*:Kokytos:8133:/
+627.1 "Beastly Roar" sync / 1[56]:[^:]*:Kokytos:8138:/
+629.2 "Beastly Bile" sync / 1[56]:[^:]*:Kokytos:813D:/
+631.2 "--sync--" sync / 1[56]:[^:]*:Kokytos:813E:/
+638.3 "Pulverizing Pounce" sync / 1[56]:[^:]*:Kokytos:813F:/
+640.4 "--sync--" sync / 1[56]:[^:]*:Kokytos:8140:/
+643.5 "--sync--" sync / 1[56]:[^:]*:Kokytos:8140:/
+653.6 "Gluttonous Rampage" sync / 1[56]:[^:]*:Kokytos:8139:/
+653.8 "Ravenous Bite 1" sync / 1[56]:[^:]*:Kokytos:813A:/
+655.9 "Ravenous Bite 2" sync / 1[56]:[^:]*:Kokytos:813A:/
+658.1 "Touchdown" sync / 1[56]:[^:]*:Kokytos:8137:/
```

Comment:
This is a proximity AoE centered on the centre of the arena, is there some trigger message I can reuse for that?

Reply:
I don't think we have a dedicated response for this. A custom "get to edge" trigger would fit with previous similar mechanics, or maybe `responses.getOut()` would fit if the damage falloff is linear. (If it's not, such as with the P4S Lightning, we would want to be explicit that the user has to be *all* the way out.)

Reply:
In a general sense, I feel like players are probably looking at their feet (i.e. don't stand in telegraphs, move away from proximity markers) and haven't written triggers for these.  I think if you wanted to add a trigger for this, I agree  `Responses.getOut()` seems fine to me for this one.

Reply:
I guess the proximity marker there is pretty standard and you have loads of time to resolve it (and many ways to do so, as far as I can tell the tankbusters at the same time don't even cleave), so I'll probably leave it without a trigger.